packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/common_use_cases/refresh_access_token_use_case.dart

```diff
@@ -0,0 +1,25 @@
+import '../repositories/auth_repository.dart';
+
+/// Obtains a new access token from the remote API
+/// using the current refresh token.
+class RefreshAccessTokenUseCase {
```

Comment:
General note: For most of the projects the preferred way of handling the domain business logic is with services. Do you think we need to create a ticket for converting all use cases to services ?